packages/loot-core/src/shared/util.js

```diff
@@ -300,7 +296,7 @@ export function getNumberFormat() {
   return numberFormat;
 }
 
-setNumberFormat('1,000.33');
+setNumberFormat('comma-dot');
```

Comment:
Passing the format name like we do everywhere else. 
Passing an example number string simply falls through to the switch default case, so this is no change.

Reply:
Yeah, I've noticed a couple of places where we're passed invalid variables into places. Ideally we can clean those up as part of the TS migration.